frontend/app/moveFunctions.js

```diff
@@ -0,0 +1,82 @@
+// functions to move around the board
+// will have to change coordinates as browser interpret these differently.
+// only test coordinates for now.
+export function getCoord(position) {
+    const listOfCoordinates = {
+        0: [735, 720],
+        1: [640, 720],
+        2: [578, 720],
+        3: [512, 720],
+        4: [444, 720],
+        5: [380, 720],
+        6: [310, 720],
+        7: [250, 720],
+        8: [183, 720],
+        9: [119, 720],
+        10: [15, 720],
```

Comment:
This 10 should be closer to 11 and a line away from 9 :grinning: because there's 11 positions in this group, assuming the space between the groups of co-ordinates represents each side of the board (being picky, sorry).

Reply:
Good point Arash!